src/pathways/escher.service.spec.ts

```diff
@@ -0,0 +1,90 @@
+// import {} from 'jasmine';
+
+import { EscherService } from './escher.service';
+
+describe('EscherService.alignReactions', () => {
```

Comment:
You mean referencing the service would be better like this? I agree that it's more readable, but on the other hand text search won't show it. What do you think?